pdns/dnssecinfra.hh

```diff
@@ -189,6 +194,7 @@ class DNSCryptoKeyEngine
       static allmakers_t s_allmakers;
       return s_allmakers;
     }
+    static std::unordered_set<unsigned int> s_switchedOff;
```

Comment:
Perhaps we could add a comment stating that this can only be altered at configuration time, when no other thread can touch it? Perhaps one day we could even automatically enforce that with a wrapper class that refuses to provide write access once the configuration phase is over?

Reply:
done